wfe2/wfe.go

```diff
@@ -492,31 +493,40 @@ func (wfe *WebFrontEndImpl) NewAccount(
 		return
 	}
 
-	var init core.Registration
+	type accountCreateRequest struct {
```

Comment:
nit: since this isn't referred to by name anywhere else this could just be `var init struct ...`.